fiftyone/__init__.py

```diff
@@ -9,6 +9,22 @@
 """
 
 from pkgutil import extend_path as _extend_path
+from sys import hexversion
+from os import getenv
+
+# Python 3.8 goes EoL in October, 2024
+#  We should tell folks we won't support those Python versions after 9/24
+
+PYTHON_38_NOTICE = getenv(
+    'FIFTYONE_PYTHON_38_DEPRECATION_NOTICE', "True"
+) == "True"
+
+if hexversion < 0x30900f0 and hexversion >= 0x30800f0 and PYTHON_38_NOTICE:
+    print("***Python 3.8 Deprecation Notice***")
```

Comment:
nit: suggest using `logger.info()` rather than `print()`, for those users who still want the notice, but want it piped to their configured logger

https://github.com/voxel51/fiftyone/blob/develop/STYLE_GUIDE.md#logging

Reply:
I don't really like all the noise that comes with `logger`, but sure:
```
root@65a3b4743cf9:/# python -c 'import fiftyone as fo'
WARNING:root:***Python 3.8 Deprecation Notice***
WARNING:root:Python 3.8 will no longer be supported in new releases afterOctober 1, 2024.
WARNING:root:Please upgrade to Python 3.9 or later.
WARNING:root:For additional details please see https://deprecation.voxel51.com
root@65a3b4743cf9:/# export FIFTYONE_PYTHON_38_DEPRECATION_NOTICE=false
root@65a3b4743cf9:/# python -c 'import fiftyone as fo'
root@65a3b4743cf9:/#
```